requirements.txt

```diff
@@ -32,18 +32,21 @@ freetype-py==1.0
 sh==1.08
 pypng==0.0.17
 
-# Additional dependencies
-kombu==3.0.26
-mercurial==3.3.3
-oauthlib==0.7.2
-Padding==0.4
+# Additional requirements
+anyjson==0.3.3
+billiard==3.3.0.23
+django-celery==3.1.17
```

Comment:
Thanks! Will fix alongside the conflict resolution. These duplicates keep creeping in when I merge things.